zulipterminal/cli/run.py

```diff
@@ -343,13 +349,19 @@ def _write_zuliprc(
 
 def check_for_default_zuliprc() -> str:
     zuliprc_locations = [
-        CONFIG_PATH_ZULIPRC,
         DOWNLOADED_PATH_ZULIPRC,
         HOME_PATH_ZULIPRC,
     ]
     valid_locations = [
         location for location in zuliprc_locations if Path(location).exists()
     ]
+    valid_locations.extend(
+        str(file) for file in Path(ZULIP_CONFIG_PATH).glob("zuliprc")
+    )
+    valid_locations.extend(
```

Comment:
The current implementation explicitly does not use `Path(ZULIP_CONFIG_PATH).glob("**/zuliprc")`, to avoid the situation (in later commits) where the realm has multiple zuliprc files inside it (in folders or as loose file). This implementation will only recognize the outermost zuliprc within a realm's directory, irrespective of the subdirectories' contents.